src/rules/compositeFieldsAreNullable.ts

```diff
@@ -19,9 +20,11 @@ import {
 function isAllowedNonNullCompositeField(
   node: FieldDefinitionNode,
   type: GraphQLCompositeType,
+  exceptions: Array<string>,
 ): boolean {
-  // allowlist
-  if (type.name === 'PageInfo') {
+  // exceptions
+  const isException = exceptions.indexOf(type.name) > -1;
```

Comment:
I think it's possible to use `Array.includes` here:

```suggestion
  const isException = exceptions.includes(type.name);
```